### PR TITLE
refreshed some of the language around pull request reviews and differ…

### LINKED ISSUE #14874: Clarify language around permissions for PR reviews
NOTE This issue is reserved for our MLH fellowship program.

### What article on docs.github.com is affected?

https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/requesting-a-pull-request-review

### What part(s) of the article would you like to see updated?

We received the following feedback comment:

> What does it mean when there is no gear icon next to "Reviewers"? What permissions do I need to request for this repo

There is a note box that highlights who has permission, but this might not be immediately noticeable to a reader who is using the procedural. The first two paragraphs and note are a little confusing because they describe the permissions necessary for user, org, and repos.

We should clearly describe what permissions are needed for each level. Ideally use simple language to ensure that the reader is immediately clear. 

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by cmwilson21**
…ent account types

### Why:
The wording around who has access to request a PR review was a bit confusing, per feedback mentioned in the issue. 
Closes [14874](https://github.com/github/docs/issues/14874)

### What's being changed:
I added some lines differentiating GitHub roles and (hopefully) clarifying some of the confusion around who can request a pull request review and who has to have a review assigned to their request.

### Check off the following:

- [x] I have reviewed my changes in staging (look for "Automatically generated comment" and click **Modified** to view your latest changes).
- [x] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/contributing/self-review.md#self-review).

### Writer impact (This section is for GitHub staff members only):

- [ ] This pull request impacts the contribution experience
  - [ ] I have added the 'writer impact' label
  - [ ] I have added a description and/or a video demo of the changes below (e.g. a "before and after video")

### CODE REVIEW

**Comment by ramyaparimi** [comment]
@cmwilson21 
Thanks so much for opening a PR! I'll get this triaged for review :zap:

File: content/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/requesting-a-pull-request-review.md [hunk 1]
```diff
@@ -16,9 +16,11 @@ topics:
...
+In order to request a pull request review, you will need [write access](/organizations/managing-access-to-your-organizations-repositories/repository-roles-for-an-organization) permissions. Conversely, owners or collaborators can assign a pull request review to any person that has been granted [read access](/get-started/quickstart/github-glossary#read-access) only.
+
+Organization members can also assign a pull request review to any person or team with read access to a repository. The requested reviewer or team will receive a notification that you asked them to review the pull request. {% ifversion fpt or ghae or ghes or ghec %}If you request a review from a team and code review assignment is enabled, specific members will be requested and the team will be removed as a reviewer. For more information, see "[Managing code review settings for your team](/organizations/organizing-members-into-teams/managing-code-review-settings-for-your-team)."{% endif %}
```
Comment (hubwriter) [inline content/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/requesting-a-pull-request-review.md:23]: Maybe, to reinforce the point, we should add that member must have write access:
```suggestion
Organization members with write access can also assign a pull request review to any person or team with read access to a repository. The requested reviewer or team will receive a notification that you asked them to review the pull request. {% ifversion fpt or ghae or ghes or ghec %}If you request a review from a team and code review assignment is enabled, specific members will be requested and the team will be removed as a reviewer. For more information, see "[Managing code review settings for your team](/organizations/organizing-members-into-teams/managing-code-review-settings-for-your-team)."{% endif %}
```

File: content/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/requesting-a-pull-request-review.md [hunk 2]
```diff
@@ -16,9 +16,11 @@ topics:
...
 
-Owners or collaborators can assign a pull request review to any person that has been explicitly granted [read access](/articles/access-permissions-on-github) to a user-owned repository. Organization members can assign a pull request review to any person or team with read access to a repository. The requested reviewer or team will receive a notification that you asked them to review the pull request. {% ifversion fpt or ghae or ghes or ghec %}If you request a review from a team and code review assignment is enabled, specific members will be requested and the team will be removed as a reviewer. For more information, see "[Managing code review settings for your team](/organizations/organizing-members-into-teams/managing-code-review-settings-for-your-team)."{% endif %}
+In order to request a pull request review, you will need [write access](/organizations/managing-access-to-your-organizations-repositories/repository-roles-for-an-organization) permissions. Conversely, owners or collaborators can assign a pull request review to any person that has been granted [read access](/get-started/quickstart/github-glossary#read-access) only.
```
Comment (hubwriter) [inline content/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/requesting-a-pull-request-review.md:21]: EFL readers are sometimes confused by "In order", as it can suggest that ordering is involved. I'd drop this. 

Again, I'd also use a "For more information..." link here.

"Conversely" normally reverses the previous statement, which I don't think it does here. The mention of "owners or collaborators" (from the original) is a little misleading here, I think, because only collaborators with write access can assign reviewers. It's the write access (which you've added in the first sentence) that's the important thing. I'd reword this a little to clarify this.

```suggestion
To assign a reviewer to a pull request, you will need write access to the repository. For more information about repository access, see "[Repository roles for an organization](/organizations/managing-access-to-your-organizations-repositories/repository-roles-for-an-organization)." If you have write access, you can assign anyone who has read access to the repository as a reviewer.
```

**Review by hubwriter — COMMENTED** [review]
Thanks for making these changes. 👍 

I've made a few suggestions.

File: content/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/requesting-a-pull-request-review.md [hunk 3]
```diff
@@ -16,9 +16,11 @@ topics:
   - Pull requests
 shortTitle: Request a PR review
 ---
-Owners and collaborators on a repository owned by a user account can assign pull request reviews. Organization members with triage permissions to a repository can assign a pull request review.
+Repositories belong to a user account (a single individual owner) or an organization account (a shared account with numerous collaborators or maintainers). You can read more about the different account types [here](/get-started/learning-about-github/types-of-github-accounts). Owners and collaborators on a repository owned by a user account can assign pull request reviews. Likewise, organization members with triage permissions can also assign a pull request.
```
[the remark this answers is not in the document]
Comment (cmwilson21) [reply content/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/requesting-a-pull-request-review.md:19]: This all makes perfect sense. I didn't really like the "here" link, either, so thank you for the suggestion and I will pocket it for future use.

File: content/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/requesting-a-pull-request-review.md (hunk 2 above)
Comment (cmwilson21) [reply content/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/requesting-a-pull-request-review.md:21]: Again, thank you for the clarification and the tip on the EFL reader.

File: content/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/requesting-a-pull-request-review.md (hunk 1 above)
Comment (cmwilson21) [reply content/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/requesting-a-pull-request-review.md:23]: I agree with this, too. Thank you for taking the time to make such great notes. I will work on all this for future issues.

File: content/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/requesting-a-pull-request-review.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (adwokat) [reply content/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/requesting-a-pull-request-review.md]: Привіт
Скачайте

**Review by hubwriter — APPROVED** [review]
LGTM 👍 
Thanks for making this change. 🎖️
